.github/workflows/build.yml

```diff
@@ -81,3 +81,20 @@ jobs:
             branch: gh-pages
             force: true
             directory: .
+
+      - name: Commit haskell package
+        if: github.ref == 'refs/heads/master'
+        run: |
+            nix-build -A ledger.hsSrc -j1 -o outputs/ledgerSrc
+            git checkout master
+            git add -f outputs/ledgerSrc
+            git commit -m "Updated for ${{ github.sha }}"
```

Comment:
Does this mean we're going to store the MAlonzo code in this repo from now on?

Reply:
Yes